Meadowlark-js/packages/meadowlark-core/src/security/JwtValidator.ts

```diff
@@ -20,19 +20,14 @@ export function validateJwt(authorizationHeader: string | undefined): JwtValidat
   }
 
   if (jwtStatus.isMissing) {
-    if (jwtStatus.isOwnershipEnabled) {
-      return {
-        jwtStatus,
-        errorResponse: {
-          statusCode: 401,
-          body: '{ "error": "invalid_client", "error_description": "Authorization token not provided" }',
-          headers: { 'WWW-Authenticate': 'Bearer' },
-        },
-      };
-    }
-
-    // Ownership security is off
```

Comment:
With this being removed, please clean up `.http` files where appropriate: removing anything whose sole purpose is to test the "no auth", and adding the auth header to anything that is useful and doesn't have it yet.

Reply:
👍